src/backend/distributed/planner/multi_logical_optimizer.c

```diff
@@ -2013,6 +2013,10 @@ MasterAggregateExpression(Aggref *originalAggregate,
 			elog(ERROR, "Aggregate lacks COMBINEFUNC");
 		}
 	}
+	else if (aggregateType == AGGREGATE_CUSTOM_NO_PUSHDOWN)
```

Comment:
This doesn't make sense. We need the aggregate to be applied to every record, even if the parameters are constant